### ISSUE TITLE
Can we pick a semi-permanent new default popup height?

### ISSUE THREAD

**Issue by jaredhirsch** [mozilla/universal-search-addon#127]
Per discussion in #105, we might not be unloading the popup CSS file before mozlando.

That CSS file mainly sets the `-moz-binding` needed to attach our popup to the XUL DOM, but it also sets the default height of the popup; so if we change the popup's height as part of an update, the initial render might be the wrong height in already-opened windows after an update.

However, we're planning to let the iframe control the height before Mozlando (#19), so this might only result in a brief initial flicker in the case where (1) your addon updated without a restart, (2) the update included a change to the default height of the popup, (3) and only the already-opened windows would be affected.

TL;DR: The simplest approach would be to pick a semi-permanent default iframe height, whether that's the current height, or something taller or shorter, and plan for a slightly janky upgrade whenever we adjust that height in the future. Can we live with this for Mozlando? I think so / hope so.

**Comment by jaredhirsch** [comment]
Now that #148 has landed, this bug is obsolete: we set a very, very small min-height in the CSS file, and rely on the iframe to manage the popup's height.
